blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
But [here]( https://github.com/bartvm/blocks/blob/master/blocks/extensions/__init__.py#L59) it is already `callback_name`. Which location in the code are you talking about?